src/admin/controllers/activities.php

```diff
@@ -1,18 +1,18 @@
 <?php
 /**
- * @version    SVN: <svn_id>
  * @package    ActivityStream
  * @author     Techjoomla <extensions@techjoomla.com>
  * @copyright  Copyright (c) 2009-2017 TechJoomla. All rights reserved.
  * @license    GNU General Public License version 2 or later.
  */
 // No direct access to this file
 defined('_JEXEC') or die;
+use Joomla\Utilities\ArrayHelper;
 
 /**
- * HelloWorlds Controller
+ * Activity Stream Controller
  *
- * @since  0.0.1
+ * @since  1.0.2
```

Comment:
@Mayaank12 
since version should be 0.0.1

Reply:
Done